src/components/CaptureRoutes/CaptureRoutes.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from "react";
+import { Outlet, useLocation } from "react-router-dom";
+
+import useAnalytics from "hooks/useAnalytics";
+
+const CaptureRoutes = (): JSX.Element => {
+  const sendAnalytics = useAnalytics();
+  const location = useLocation();
+
+  useEffect(() => {
+    // Send an analytics event when the URL changes.
+    sendAnalytics({
```

Comment:
This is now in its own components as it needs to be rendered inside a route as it uses `useLocation`,